### PR TITLE
Update diffy and git gems

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by ekohl** [comment]
Probably yes. I restarted that build because I'm wondering why it failed.

**Comment by evgeni** [comment]
Done

File: packages/plugins/rubygem-git/rubygem-git.spec [hunk 1]
```diff
@@ -77,11 +55,11 @@ cp -a .%{gem_dir}/* \
...
 %exclude %{gem_instdir}/.yardopts
-%{gem_instdir}/ISSUE_TEMPLATE.md
+%doc %{gem_instdir}/ISSUE_TEMPLATE.md
 %license %{gem_instdir}/LICENSE
-%{gem_instdir}/MAINTAINERS.md
-%{gem_instdir}/PULL_REQUEST_TEMPLATE.md
-%{gem_instdir}/RELEASING.md
+%doc %{gem_instdir}/MAINTAINERS.md
+%doc %{gem_instdir}/PULL_REQUEST_TEMPLATE.md
+%doc %{gem_instdir}/RELEASING.md
```
Comment (ekohl) [inline packages/plugins/rubygem-git/rubygem-git.spec:62]:
```suggestion
%exclude %{gem_instdir}/PULL_REQUEST_TEMPLATE.md
%exclude %{gem_instdir}/RELEASING.md
```

File: packages/plugins/rubygem-git/rubygem-git.spec [hunk 2]
```diff
@@ -77,11 +55,11 @@ cp -a .%{gem_dir}/* \
 %exclude %{gem_instdir}/.github
 %exclude %{gem_instdir}/.gitignore
 %exclude %{gem_instdir}/.yardopts
-%{gem_instdir}/ISSUE_TEMPLATE.md
+%doc %{gem_instdir}/ISSUE_TEMPLATE.md
```
Comment (ekohl) [inline packages/plugins/rubygem-git/rubygem-git.spec:58]:
```suggestion
%exclude %{gem_instdir}/ISSUE_TEMPLATE.md
```

**Review by ekohl — APPROVED** [review]

**Pull request merged**